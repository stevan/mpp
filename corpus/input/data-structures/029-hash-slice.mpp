@hash{qw(a b c)};