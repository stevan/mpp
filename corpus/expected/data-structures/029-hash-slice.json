[
  {
    "type": "HashSlice",
    "base": {
      "type": "Variable",
      "name": "@hash"
    },
    "keys": {
      "type": "List",
      "elements": [
        {
          "type": "String",
          "value": "a"
        },
        {
          "type": "String",
          "value": "b"
        },
        {
          "type": "String",
          "value": "c"
        }
      ]
    }
  }
]